aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -344,7 +344,7 @@ Tensor sparse_mask_sparse_compressed(
   }
 
   if (!mask.numel() || !mask._nnz()) {
-    return mask.clone().to(self.device(), self.scalar_type());
+    return mask.to(self.device(), self.scalar_type(), false, true);
```

Comment:
Can we just use kwargs here instead of overriding the default args?

Reply:
This is C++ code. We can use only comments.